docs/data-layer.md

```diff
@@ -0,0 +1,178 @@
+# Data Layer
+
+## Redux
+
+In eHoldings, data is connected on `routes` only. This is done via
```

Comment:
it'd be `data are` as `data` is woefully plural.  also you might wanna chuck in a link to the redux `connect` docs